docs/doc-governance.md

```diff
@@ -0,0 +1,22 @@
+# Governance
```

Comment:
Why not simplify the naming of the file by `governance.md` ?

Reply:
https://github.com/ambanum/OpenTermsArchive/pull/971#discussion_r1040359603